aws/alarms/cloudwatch.tf

```diff
@@ -58,6 +58,20 @@ resource "aws_cloudwatch_metric_alarm" "ELB_5xx_error_warn" {
   alarm_actions       = [var.sns_topic_alert_warning_arn]
 }
 
+resource "aws_cloudwatch_metric_alarm" "UnHealthyHostCount" {
+  alarm_name          = "UnHealthyHostCount-SEV1" # SEV1 will prompt the on-call team to respond.
```

Comment:
Unless you have changed this and I missed it during the pull request review, the notification lambda parses what is in the `alarm_description` so you may have to put the `SEV1` flag in there instead of the `alarm_name`

Reply:
The `getSNSMessageSeverity` string lookup checks the entire SNS message, including the alarm name.

Alarm message example: 
![image](https://github.com/cds-snc/forms-terraform/assets/70960477/35be60b0-81a4-45a7-9dc2-37b467d5ac3c)

Reply:
Oh yeah that is correct!